usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -127,12 +127,11 @@ def _get_status_url(self, file_name: str) -> str:
         if settings.IS_LOCAL:
             protocol = "http"
             host = "localhost:8000"
-        elif settings.DOWNLOAD_ENV == "production":
-            protocol = "https"
-            host = "api.usaspending.gov"
         else:
             protocol = "https"
-            host = f"{settings.DOWNLOAD_ENV}-api.usaspending.gov"
+            host = f"api.{settings.SERVER_BASE_URL}"
+            if settings.DOWNLOAD_ENV != "production":
```

Comment:
Usually we use "prod" as the code for the Production environment. It's definitely full "production" here?

Reply:
Looking at the current ansible vars for the production environment we use "production" instead of "prod".